ODP-Runtime.md

```diff
@@ -124,61 +121,54 @@ Compliance
-> Comment (SCG): A clarifying example here would be useful
+-   An ODP Platform MUST keep the same basic directory layout and content as the equivalent Apache component. Changes to that directory layout MUST be enabled by the component itself with the appropriate configurations for that layout configured.  For example, if Apache Hadoop YARN's package distribution contains a share directory with content, then that share directory with the equivalent content must be preset.
 
--   An ODP Platform MUST set the HADOOP\_COMMON\_HOME, HADOOP\_HDFS\_HOME, HADOOP\_MAPRED\_HOME, and HADOOP\_YARN\_HOME to an absolute directory of that component's location.  HADOOP\_COMMON\_LIB\_JARS\_DIR, HDFS\_LIB\_JARS\_DIR, MAPRED\_LIB\_JARS\_DIR, and YARN\_LIB\_JARS\_DIR MUST be set the relative directory to the jars of that associated component's \_HOME variable. (See [*this document*](https://github.com/apache/hadoop/blob/0bc15cb6e60dc60885234e01dec1c7cb4557a926/hadoop-common-project/hadoop-common/src/main/bin/hadoop-layout.sh.example) for related Apache Hadoop documentation.) For example:
+-   An ODP Platform MUST set the `HADOOP_COMMON_HOME`, `HADOOP_HDFS_HOME`, `HADOOP_MAPRED_HOME`, and `HADOOP_YARN_HOME` to an absolute directory of that component's location.  `HADOOP_COMMON_LIB_JARS_DIR`, `HDFS_LIB_JARS_DIR`, `MAPRED_LIB_JARS_DIR`, and `YARN_LIB_JARS_DIR` MUST be set the relative directory to the jars of that associated component's `_HOME` variable. (See [*this document*](https://github.com/apache/hadoop/blob/0bc15cb6e60dc60885234e01dec1c7cb4557a926/hadoop-common-project/hadoop-common/src/main/bin/hadoop-layout.sh.example) for related Apache Hadoop documentation.) For example:
 
 ```bash
 HADOOP_COMMON_HOME="/usr/lib/hadoop-common"
 HADOOP_COMMON_LIB_JARS_DIR="share/hadoop/common/lib"
 ```
 
-This enables applications the capability to locate where the various Apache Hadoop components are located (user-level binaries and Java JAR files).  The content of these LIB\_JARS\_DIR directories MUST be the same as the ODP Reference Implementation and the Apache Hadoop distribution.
+This enables applications the capability to locate where the various Apache Hadoop components are located (user-level binaries and Java JAR files).  The content of these `LIB_JARS_DIR` directories SHOULD be the same as the ODP Reference Implementation and the Apache Hadoop distribution of the appropriate platform.  In a future release, this will become a MUST.
+
+-   The location of the tools jar and other miscellaneous should be set to the `HADOOP_TOOLS_PATH` environment variable.  Unlike the other  `LIB_JARS_DIR` environment variables, this MUST be an absolute path and MAY contain additional content. The entire directory SHOULD NOT, by default, be included in the default hadoop class path.  Individual jars MAY be specified, however. [**TODO: Update HADOOP-10787.**]
```

Comment:
While working on HADOOP-10787, I want to update this to say that HADOOP_TOOLS_PATH has the same characteristics as CLASSPATH/HADOOP_CLASSPATH.  This makes a lot of things easier.